src/Cassandra/Metadata.cs

```diff
@@ -41,12 +40,15 @@ public class Metadata : IDisposable
         private volatile TokenMap _tokenMap;
         private volatile ConcurrentDictionary<string, KeyspaceMetadata> _keyspaces = new ConcurrentDictionary<string, KeyspaceMetadata>();
         private volatile SchemaParser _schemaParser;
-        private volatile ConcurrentDictionary<string, long> _keyspacesLastUpdateTicks = new ConcurrentDictionary<string, long>();
-        private long _lastRebuildTicks = 0;
+        private volatile ConcurrentDictionary<string, long> _keyspacesUpdateCounters = new ConcurrentDictionary<string, long>();
```

Comment:
No need for `volatile` access, use `readonly` instead.

Reply:
True! changed